.github/workflows/maven.yaml

```diff
@@ -44,6 +44,12 @@ jobs:
         with:
           java-version: 21
           distribution: 'temurin'
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.13.1
+        with:
+          minikube version: 'v1.35.0'
+          kubernetes version: 'v1.32.0'
```

Comment:
I wonder if we should be paramaterizing this so we can test against a variety of Kube versions.